src/css/components/Footer.scss

```diff
@@ -0,0 +1,30 @@
+@import '../coa_variables';
```

Comment:
I feel pretty ok about that. I hesitated to do it the way I did since you hadn't created a components scss folder yet. I was thinking about floods pain too.